pom.xml

```diff
@@ -573,6 +573,24 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.srcclr</groupId>
+                <artifactId>srcclr-maven-plugin</artifactId>
+                <version>2.0.4</version>
+                <configuration>
+                    <apiToken>
+                        eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Mjc2NSwiZXhwIjoxNDk1OTIwMDMyLCJhdXRob3JpdGllcyI6IkFHRU5UIiwianRpIjoiYzlmZTg2NDEtYzg1Ny00OTg1LTg0YjgtNjZmMjQ3ZGJjYTc1IiwiY2xpZW50X2lkIjoiIiwic2NvcGUiOltdfQ.H60FBBY8lh2QdfN1iUVwDYxSiVjTPkXuF6IZSezy4DBYO6vvyl0HFDTjv7VYX77fSf9Wj5Y0FbCizK8knbKSfurG6uAaZV3he5r2QahIT4PEV6ayYZy0Z9vJAgJ33iRSXlRVmPkX4qHyAAyCRwxX26aIg4_uUg3YdFzz_pNcWT0
```

Comment:
Sorry, updated description to be more clear. I found out that we can customize the permissions on that token so it is only valid for our codice/ddf repo. This makes it safe to commit, since it can't be misused by anyone else now.